src/main/java/org/gitlab4j/api/JobApi.java

```diff
@@ -139,8 +139,24 @@ public Stream<Job> getJobsStream(Object projectIdOrPath, JobScope scope) throws
      * @throws GitLabApiException if any exception occurs during execution
      */
     public List<Job> getJobsForPipeline(Object projectIdOrPath, long pipelineId) throws GitLabApiException {
-        Response response = get(Response.Status.OK, getDefaultPerPageParam(),
-                "projects", getProjectIdOrPath(projectIdOrPath), "pipelines", pipelineId, "jobs");
+        return getJobsForPipeline(projectIdOrPath, pipelineId, false);
+    }
+
+    /**
+     * Get a list of jobs in a pipeline.
+     *
+     * <pre><code>GitLab Endpoint: GET /projects/:id/pipelines/:pipeline_id/jobs</code></pre>
+     *
+     * @param projectIdOrPath id, path of the project, or a Project instance holding the project ID or path to get the pipelines for
+     * @param pipelineId the pipeline ID to get the list of jobs for
+     * @param includeRetried the includeRetried to get also retried jobs
+     * @return a list containing the jobs for the specified project ID and pipeline ID
+     * @throws GitLabApiException if any exception occurs during execution
+     */
+    public List<Job> getJobsForPipeline(Object projectIdOrPath, long pipelineId, boolean includeRetried) throws GitLabApiException {
+        GitLabApiForm formData = new GitLabApiForm().withParam("include_retried", includeRetried).withParam(PER_PAGE_PARAM, getDefaultPerPage());
+        Response response = get(Response.Status.OK, formData.asMap(),
+            "projects", getProjectIdOrPath(projectIdOrPath), "pipelines", pipelineId, "jobs");
         return (response.readEntity(new GenericType<List<Job>>() {}));
```

Comment:
Here I would just delegate to the other method:

```suggestion
        return getJobsForPipeline(projectIdOrPath, pipelineId, null, includeRetried)
```